vmdb/app/controllers/application_controller.rb

```diff
@@ -573,6 +573,10 @@ def render_flash(add_flash_text = nil, severity = nil)
     end
   end
 
+  def tagging_explorer_controller?
```

Comment:
@tenderlove Can you comment on the use of inheritance here for determining functionality?  It's a *much* nicer approach than what was previous done, but is it the best/right approach.